### PR TITLE
libraries/webkit2gtk4.1: Added vala bindings

### PR DESCRIPTION
**Pull request by willysr**

### CODE REVIEW

**Comment by willysr** [comment]
@sbo-bot: build libraries/webkit2gtk4.1

File: libraries/webkit2gtk4.1/webkit2gtk4.1.SlackBuild [hunk 1]
```diff
@@ -114,6 +115,16 @@ mkdir -p $PKG/usr/share/gtk-doc/html/webkit{2,dom}gtk-4.1
...
 
+#add vala bindings
+mkdir -p $PKG/$VALADIR
+echo -e "gtk+-3.0\njavascriptcoregtk-4.1\nlibsoup-3.0\n">$PKG/$VALADIR/webkit2gtk-4.1.deps
+chmod 644 $PKG/$VALADIR/webkit2gtk-4.1.deps
+cp -a $PKG/$VALADIR/webkit2gtk-4.1.deps $PKG/$VALADIR/webkit2gtk-web-extension-4.1.deps
+cp -a $CWD/webkit2gtk-4.1.vapi $PKG/$VALADIR/webkit2gtk-4.1.vapi
+cp -a $CWD/webkit2gtk-web-extension-4.1.vapi $PKG/$VALADIR/webkit2gtk-web-extension-4.1.vapi
+cp -a $CWD/javascriptcoregtk-4.1.vapi $PKG/$VALADIR/javascriptcoregtk-4.1.vapi
+chmod 644 $PKG/$VALADIR/webkit2gtk-4.1.vapi $PKG/$VALADIR/webkit2gtk-web-extension-4.1.vapi $PKG/$VALLADIR/javascriptcoregtk-4.1.vapi
```
[the remark this answers is not in the document]
Comment (timsoftgit) [reply libraries/webkit2gtk4.1/webkit2gtk4.1.SlackBuild:126]: fix typo. (remove extra L n VALADIR )

**Comment by willysr** [comment]
@sbo-bot: build libraries/webkit2gtk4.1

**Comment by willysr** [comment]
merged

**Pull request closed without merging**